its/projects/ProjectWithSourceGenerator/ProjectWithSourceGenerator.sln

```diff
@@ -0,0 +1,31 @@
+﻿
```

Comment:
maybe name the solution and top folder differently, to make it obvious what tests it's being used in?

because `its/projects/DuplicateAnalyzerReferences/` is also a `ProjectWithSourceGenerator` solution...